backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/UpdateVmCommand.java

```diff
@@ -1176,6 +1176,12 @@ protected boolean validate() {
             return failValidation(EngineMessage.VM_MIGRATION_IS_NOT_SUPPORTED);
         }
 
+        final Integer parallelMigrations = vmFromParams.getParallelMigrations();
+        if (parallelMigrations != null
```

Comment:
I see in the database that the parallel_migrations field can be null - would it make sense to change the field to be not-null and defaults to 0? because then we could check for null *or* invalid value

Reply:
I'm not sure I understand what you mean exactly.  The value can be null here, it means the value is inherited from cluster and it is also the default for VMs.  But I added a check for non-null value in clusters.